### PR TITLE
update file path for image at end of contributing.md

### PR DESCRIPTION
**Pull request by jukent**
Closes issue #149

### CODE REVIEW

**Comment by jukent** [comment]
Just realized that I accidentally made this branch off of the upstream main instead of my own fork, sorry for any hiccups this causes.

**Review by kmpaul — DISMISSED** [review]
Looks good!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,4 +84,4 @@ You can re-activate the `pythia` conda environment at any time with `conda activ
 
 Once a Pull Request has passed all tests, including the Read the Docs build, you can click "Details" of the check that says, "docs/readthedocs.org:projectpythia - Read the Docs build succeeded!" to launch a new tab with a build of the Project Pythia site. (You may have to click "Show all checks" for this to be displayed.)
 
-![Checks](../_static/images/ReadtheDocsAutobuild.png)
+![Checks](/content/_static/images/ReadtheDocsAutobuild.png)
```
[the remark this answers is not in the document]
Comment (jukent) [reply CONTRIBUTING.md]: Yes I think that's right.

**Review by brian-rose — APPROVED** [review]

**Review by kmpaul — APPROVED** [review]
Good catch, @brian-rose!  And thanks for the PR, @jukent!

**Pull request merged**